testing/camunda-process-test-example/src/test/java/io/camunda/ProcessUnitTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda;
+
+import static io.camunda.process.test.api.CamundaAssert.assertThat;
+
+import io.camunda.process.test.api.CamundaProcessTestContext;
+import io.camunda.process.test.api.CamundaSpringProcessTest;
+import io.camunda.zeebe.client.ZeebeClient;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(
+    properties = {
+      "camunda.client.zeebe.defaults.enabled=false" // disable all job workers
+    })
```

Comment:
💭 I don't think it's straightforward to do. But it would be cool if we could configure this with a property on the `CamundaSpringProcessTest` annotation. It makes it a lot easier for users to control versus this obscure setting they likely won't know exists 😄

Reply:
I'm not sure about this. :thinking: This configuration is part of the Spring SDK and documented here: https://docs.camunda.io/docs/next/apis-tools/spring-zeebe-sdk/configuration/#disable-worker. 

Let's wait for the user feedback. I'll keep this option in mind.